.asf.yaml

```diff
@@ -31,16 +31,12 @@ github:
   enabled_merge_buttons:
     squash: true
     merge: false
-    rebase: false
+    rebase: true
```

Comment:
We can use rebase to some PRs collaborations with multiple people.

Reply:
I'm fine with this.